charts/redis-ha/templates/redis-ha-statefulset.yaml

```
@@ -263,6 +256,13 @@ spec:
         - redis-server
         args:
         - /data/conf/redis.conf
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+              - ALL
```

No indent sequences?

```suggestion
            - ALL
```